src/styles/fonts.scss

```diff
@@ -0,0 +1,69 @@
+$font-folder: "../../fonts";
```

Comment:
Can you add `!default` here so that [it can be overridden from outside](https://sass-lang.com/documentation/variables/#default-values)?